backend/core/src/migration/1639051722043-add-mfa-related-columns-to-user-entity.ts

```diff
@@ -10,7 +10,7 @@ export class addMfaRelatedColumnsToUserEntity1639051722043 implements MigrationI
     await queryRunner.query(`ALTER TABLE "user_accounts" ADD "mfa_code" character varying`)
     await queryRunner.query(`
         UPDATE user_accounts
-        SET mfa_enabled = true
+        SET mfa_enabled = false
```

Comment:
do we need a new migration so this runs?

Reply:
No, I already updated detroit dev to set it to false, and when this runs on production it'll be as we want it.